### PR TITLE
New icon for Ciborium

### LINKED ISSUE #2: Use a suru icon
The current icon stands out as being very different from the rest of the icons and not in the suru style at all.

### PR DESCRIPTION
**Pull request by UniversalSuperBox**
Thanks to the work of the UBports UX and Graphics team, we now have a new icon for Ciborium! No more Humanity SD card on our homescreen!

image

It also fits right in on the Apps scope:
image

This PR, aided by #6, closes #2.

### CODE REVIEW

**Comment by UniversalSuperBox** [comment]
Note: Ciborium still uses the old Humanity icon for its "device detected" notification. We should add an icon to the Suru set as well.

File: notifications/notifications.go [hunk 1]
```diff
@@ -17,7 +17,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-// Notifications lives on a well-knwon bus.Address
+// Notifications lives on a well-known bus.Address
```
Comment (NeoTheThird) [inline notifications/notifications.go:20]: lol?

**Review by NeoTheThird — APPROVED** [review]
lgtm, good job

**Review by mariogrip — APPROVED** [review]
LGTM

**Pull request merged**